app/initializers/embedded.js

```diff
@@ -1 +1 @@
-export { default, initialize } from 'ember-cli-embedded/initializers/embedded'
+export { default, initialize } from 'ember-cli-embedded/initializers/embedded';
```

Comment:
I'm surprised by this kind of changes... Did it was intentionnal ? Did we have the pdoc linter setup in this project ? I'm surprise the linting does not complain about the `;` & no new line at the end of the file....

Reply:
PeopleDoc lint config is currently not set up in this project:
https://github.com/peopledoc/ember-cli-embedded/blob/29637ee777a2ad60439ae5b060e1df524bbd98bf/package.json#L72-L78

And because of the TypeScript layer, we would even need to implement `typescript-eslint`, but as mentioned in the initial post: not now. After this PR sure the codebase would not be in a "perfect shape", we would have some inconsistencies...

Reply:
In this context (the new `;`) it was simply to comply to the current format/lint rules set up by the new version of `ember-cli` (Prettier). I did not want to customise further the state of the PR, just a "take everything new from the recent version of `ember-cli` (so that the project runs on a solid a recent foundation), inject and renovate the code of the Initializers and the Service, done"